Release v1.2.0 (#1156)
nautobot/nautobot: nautobot/docs/additional-features/filters.md

> # Additional Filters

Reviewer: Perhaps?
```suggestion
# Additional Template Filters
```

## Introduction

Nautobot uses 2 template engines internally, Django Template and Jinja2.

> Django Template is used to render the UI pages and Jinja2 is used for all extensibility features like: computed fields, custom links etc ...

Reviewer: ```suggestion
Django Template is used to render the UI pages and Jinja2 is used for features such as computed fields, custom links, export templates, etc.
```
